modules/installation-osp-setting-worker-affinity.adoc

```diff
@@ -0,0 +1,125 @@
++
+[source,terminal]
+----
+$ ./openshift-install create manifests --dir=<installation_directory>
+----
++
+where:
++
+`installation_directory` :: Specifies the name of the directory that contains the `install-config.yaml` file for your cluster.
+
+. From the newly generated manifest files, open the `MachineSet` definition file, which is in the `manifests` directory and is named `99_openshift-cluster-api_worker-machineset-0.yaml`.
+
+. Add the property `serverGroupID` to the definition beneath the `spec.template.spec.providerSpec.value` property. For example:
++
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_ID>
+    machine.openshift.io/cluster-api-machine-role: <node_role>
+    machine.openshift.io/cluster-api-machine-type: <node_role>
+  name: <infrastructure_ID>-<node_role>
+  namespace: openshift-machine-api
+spec:
+  replicas: <number_of_replicas>
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_ID>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_ID>-<node_role>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_ID>
+        machine.openshift.io/cluster-api-machine-role: <node_role>
+        machine.openshift.io/cluster-api-machine-type: <node_role>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_ID>-<node_role>
+    spec:
+      providerSpec:
+        value:
+          apiVersion: openstackproviderconfig.openshift.io/v1alpha1
+          cloudName: openstack
+          cloudsSecret:
+            name: openstack-cloud-credentials
+            namespace: openshift-machine-api
+          flavor: <nova_flavor>
+          image: <glance_image_name_or_location>
+          serverGroupID: aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee <1>
+          kind: OpenstackProviderSpec
+          networks:
+          - filter: {}
+            subnets:
+            - filter:
+                name: <subnet_name>
+                tags: openshiftClusterID=<infrastructure_ID>
+          securityGroups:
+          - filter: {}
+            name: <infrastructure_ID>-<node_role>
+          serverMetadata:
+            Name: <infrastructure_ID>-<node_role>
+            openshiftClusterID: <infrastructure_ID>
+          tags:
+          - openshiftClusterID=<infrastructure_ID>
+          trunk: true
+          userDataSecret:
+            name: <node_role>-user-data
+          availabilityZone: <optional_openstack_availability_zone>
+----
+<1> Add the UUID of your server group here.
+
+. Optional: Back up the `manifests/99_openshift-cluster-api_worker-machineset-0.yaml file`. The installation program deletes the `manifests/` directory when creating the cluster.
+
+The installer uses the `MachineSet` definition that you modified to create compute machines within your {rh-openstack} server group. 
```

Comment:
```suggestion
When you install the cluster, the installer uses the `MachineSet` definition that you modified to create compute machines within your {rh-openstack} server group. 
```

@rlobillo Any objections to this rewording at the end rather than a sixth step?

Reply:
https://github.com/openshift/openshift-docs/pull/34157#pullrequestreview-735765300